### ISSUE TITLE
[@types/ember bug] - SafeString is not consistent

### ISSUE THREAD

**Issue by nightire** [typed-ember/ember-cli-typescript#309]
### Which package(s) does this problem pertain to?
  - [x] @types/ember
  - [x] @types/handlebars

### What are instructions we can follow to reproduce the issue?
``` typescript
import EmberObject from '@ember/object';
import { htmlSafe } from '@ember/string';
import { SafeString } from 'handlebars';

class Foo extends EmberObject {

  get bar(): SafeString {
	return htmlSafe('lorem ipsum...');
  }

}
```

### Now about that bug. What did you expect to see?
It is expected to has no errors.

### What happened instead?
Typescript will throws an error:

```
Type 'import("ember").Ember.Handlebars.SafeString' is not assignable to type 'Handlebars.SafeString'.
  Property 'toHTML' is missing in type 'SafeString'.
```

---

This is `SafeString` in @types/handlebars:

``` typescript
export class SafeString {
    constructor(str: string);
    toString(): string;
    toHTML(): string;
}
```

And this is what `htmlSafe` returns in @types/ember:

``` typescript
namespace Handlebars {
	// ...
    class SafeString {
        constructor(str: string);
        toString(): string;
    }
	// ...
}

function htmlSafe(str: string): Handlebars.SafeString;
```

Apparently, they are not consistent.

**Comment by mike-north** [comment]
@nightire fix is waiting for merge https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28914

New version `@types/ember@^3` is likely to be cut on Mon or Tues of next week

**Comment by chriskrycho** [comment]
Thanks for writing that up so carefully and thoroughly, @nightire; and thanks for tackling it so quickly, @mike-north!
